spec/system/table_builder_spec.rb

```diff
@@ -55,9 +55,11 @@ def normalize_value(content)
             | |  |  | |
           TABLE
 
-          expect(normalize_value(composer.composer_input.value)).to eq(
-            normalize_value(created_table),
-          )
+          try_until_success do
+            expect(normalize_value(composer.composer_input.value)).to eq(
+              normalize_value(created_table),
+            )
+          end
```

Comment:
applying some fix we have at other places, ideally we should have an id or name on the textarea @renato